examples/Immer/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2016-2018, Ruslan Baratov
+# All rights reserved.
+
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(immer)
+
+# DOCUMENTATION_START {
+hunter_add_package(Immer)
+find_package(Immer CONFIG REQUIRED)
+
+add_executable(boo boo.cpp)
+target_link_libraries(boo PUBLIC immer)
```

Comment:
A little bit confusing that they mix capital and lowercase again but there’s no real fix without forking

Reply:
I would say, this is a decision by the developer. It will break "compatible" if we change this in hunter to upper case like `target_link_libraries(.... PUBLIC Immer)` and the "rest of the world" use it with lowercase `target_link_libraries(... PUBLIC immer)` .... maybe I will create an issue in his project. The same issue are also in the other project from him...